nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -371,7 +371,7 @@ def change_decoding_strategy(self, decoding_cfg: DictConfig):
         logging.info(f"Changed decoding strategy to \n{OmegaConf.to_yaml(self.cfg.decoding)}")
 
     @classmethod
-    def list_available_models(cls) -> Optional[PretrainedModelInfo]:
+    def list_available_models(cls) -> List[PretrainedModelInfo]:
```

Comment:
Revert all of these

Reply:
The purpose of these changes was inconsistent type for `list_available_models` method:
- in base class it was `Optional[PretrainedModelInfo]` (`nemo.core.classes.Model`)
- in some classes there was no return (ok for this type)
- in some classes (e.g. here) a list is returned, so actual type is `List[PretrainedModelInfo]`
- in some classes also `List[PretrainedModelInfo]` was used for type annotation

So, I used `List[PretrainedModelInfo]` in case of the list was returned, and `Optional[List[PretrainedModelInfo]]` for base class (to avoid any sort of incompatibility).

I will revert quotes in all places they were used.

Should I really revert all type annotations for `list_available_models`? Can we keep the type correct and consistent everywhere?

Reply:
If I correctly understood our conversation in Slack, it's ok (there were no quotes here, actually `List` is returned)